src/handlers/notify_zulip.rs

```diff
@@ -236,6 +244,23 @@ pub(super) async fn handle_input<'a>(
     Ok(())
 }
 
+async fn get_zulip_ids(ctx: &Context, recipients: &[User]) -> String {
+    let gh_ids_fut = recipients
+        .iter()
+        .map(|recipient| async move { ctx.team.github_to_zulip_id(recipient.id).await });
+    let zulip_ids = join_all(gh_ids_fut).await;
+
+    zulip_ids
+        .iter()
+        .map(|x| {
+            let id = x.as_ref().unwrap().unwrap();
```

Comment:
This will crash the handler if either the PR author (more likely) or the assignee (less likely) do not have a Zulip ID in team. Could you please filter out users for which we could not determine the Zulip ID, instead of crashing?

Reply:
ah sorry that was a leftover I forgot to remove! Thanks for spotting it